srcpkgs/tiny/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'tiny'
+pkgname=tiny
+version=0.8.0
+revision=1
+build_wrksrc="tiny"
+build_style=cargo
+configure_args="--no-default-features --features=desktop-notifications --features=tls-native"
+hostmakedepends="pkg-config"
+makedepends="libressl-devel dbus-devel"
+short_desc="Tiny terminal IRC client"
```

Comment:
By "tiny" do you mean as an adjective and as the name of the program? If the latter I'd use "tiny" (lowercase) here0.

Reply:
We do have Descriptions always starting with a Uppercase letter, so `Tiny` is the correct one.